src/vector/faer_serial.rs

```diff
@@ -0,0 +1,181 @@
+use std::ops::{Mul, MulAssign};
+
+use faer::{unzipped, zipped};
+
+use crate::{scalar::Scalar, IndexType};
+
+use super::{Vector, VectorCommon, VectorIndex, VectorView, VectorViewMut};
+impl<'a, T: Scalar> Mul<crate::scalar::Scale<T>> for faer::ColRef<'a, f64> {
```

Comment:
I would import a lot of these to reduce repetition:
```
use crate::scalar::Scale
use faer::{ColRef, ColMut, Col}
```

Reply:
Done